docs/wiki/protocol/overview.md

```diff
@@ -0,0 +1,74 @@
+# Ethereum Overview
+
+## Network coming online and peer-to-peer computing
+
+The concept of peer-to-peer (P2P) networks, where computers communicate and share resources directly without a central authority, has a rich history dating back to the 1960s. Pioneering projects like the ARPANET project, switched on in late October 1969 became the first wide-area packet-switched network, laid the groundwork for this decentralized approach. ARPANET demonstrated the model of distributed communication, where "every participating node could request and serve content".
+
+![ARPANET-map](img/overview/ARPANET-map.jpg)
+> The map of ARPANET in the 1960s
+
+This decentralized philosophy gained significant traction in the late 1990s with the rise of file-sharing platforms like [Napster](https://en.wikipedia.org/wiki/Napster) (1999). These platforms challenged the traditional client-server model, where users relied on a central server to access data. Napster, and subsequent platforms like [BitTorrent](https://en.wikipedia.org/wiki/BitTorrent) (2001), sparked new conversations and philosophies highlighting the potential for a more democratic and distributed online interaction.
+
+## Introduction of Bitcoin
+
+Introduced in a 2008 paper by Satoshi Nakamoto titled ["Bitcoin: A Peer-to-Peer Electronic Cash System"](https://bitcoin.org/bitcoin.pdf), blockchain established a distributed ledger system where data is cryptographically linked in chronological blocks. This innovation empowered the creation of Bitcoin, the first decentralized digital currency, eliminating the need for trusted third-party intermediaries like banks.
```

Comment:
Bitcoin is built on more concepts than just p2p tech, I would mention add a section or add to previous one something on cryptography, initial attempts for digital cash by cypherpunks..

Reply:
Noted. That makes me think this page overlaps with history of Ethereum. Should we get rid of that page?

Reply:
Yes, let's keep this as Prehistory+intro and the History page should track the evolution of the protocol itself